packages/loader/src/loader.ts

```diff
@@ -81,44 +75,45 @@ export function removeNullValues(content: Record<string, unknown>): Record<strin
   return copy
 }
 
-/** @internal */
-export function toMetadata(
-  model: StreamID | string,
-  controller?: string,
-): ModelInstanceDocumentMetadataArgs {
-  return {
-    controller,
-    model: model instanceof StreamID ? model : StreamID.fromString(model),
-  }
-}
-
-const tempBatchLoadFn: BatchLoadFn<DocID, ModelInstanceDocument> = () => Promise.resolve([])
+const tempBatchLoadFn: BatchLoadFn<LoadKey, ModelInstanceDocument> = () => Promise.resolve([])
```

Comment:
IIRC this was needed because TS wouldn't allow calling `super()` with a function using `this`, but looks like it's since been fixed in TS so it's no longer needed, I updated the code.